### DISCUSSION TITLE
Migrations API broken?

### DISCUSSION THREAD

**Discussion by davidgfnet** [community/community#13851] [Apps, API and Webhooks]
Hello there!
I created migrations (as a way to backup stuff from my account) on a regular basis, but never really deleted the archive (I would just download the zip file and forget). After a while I cannot longer create a new migration (status: failed) but no human readable error message!
I tried to list the migrations I have and then proceed to delete them using the API, however the /user/migrations endpoint gives me a Server Error after a while (it looks to me like some database query is timing out internally, since I must have so many).
So TLDR I'm pretty much stuck! I'd appreciate someone from github bumping up that internal timeout or at least helping me out clean my migrations :) Not sure if this can be done via a website or what.
Thanks!
David

**Comment by davidgfnet** [comment]
I suspect that the migration creation issue is due to the size (or perhaps even an internal timeout) of the migration. Now I'm requesting partial migrations (N repos at a time) and this seems to work if I keep the migration at around 4GB or so.

**Comment by maelle** [comment]
@davidgfnet we observed the same as you for organization migrations, that it was more reliable to split the migration. It's a bit frustrating not to get an error message in case of failures.

Note that migration archives are deleted after 7 days https://docs.github.com/en/rest/migrations/users?apiVersion=2022-11-28 so you're not hoarding them if I understand correctly.

**Comment by davidgfnet** [comment]
As of today, listing the migrations returns a 500 error still. I suspect I might have created too many migrations. But if can't list them, I can't delete them.
